manage.py

```diff
@@ -118,6 +97,86 @@ def analyze_all(self):
         )
         print('Batcher invocation successful!')
 
+    @staticmethod
+    def build():
+        """Build Lambda packages (saves *.zip files in terraform/)."""
+        lambda_build(TERRAFORM_DIR)
+
+    def deploy(self):
+        """Deploy BinaryAlert. Equivalent to test + build + apply + analyze_all."""
+        self.test()
+        self.build()
+        self.apply()
+        self.analyze_all()
+
+    def live_test(self):
+        """Upload an EICAR test file to BinaryAlert which should trigger a YARA match alert."""
+        bucket_name = '{}.binaryalert-binaries.{}'.format(
+            self._config['name_prefix'].replace('_', '.'), self._config['aws_region'])
+        test_filename = 'eicar_test_{}.txt'.format(uuid.uuid4())
+        s3_identifier = 'S3:{}:{}'.format(bucket_name, test_filename)
+
+        print('Uploading EICAR test file {}...'.format(s3_identifier))
+        bucket = boto3.resource('s3').Bucket(bucket_name)
+        bucket.put_object(
```

Comment:
My general thought is there's no point in catching an exception if you aren't going to do anything with it. I would want to dump the traceback anyway (in case it's a problem with `boto3` arguments or mocking instead of S3).

That's why I print `Uploading...` beforehand, so that if an exception happens, we know at what step it occurred